code/modules/clothing/suits/jobs.dm

```diff
@@ -39,6 +39,15 @@
 	hoodtype = /obj/item/clothing/head/chaplain_hood
 	allowed = list (/obj/item/weapon/storage/bible)
 
+//Chaplain but spookier
+/obj/item/clothing/suit/storage/hoodied/chaplain_hoodie/whiteout
+	name = "white robe"
+	desc = "A long, flowing white robe. It looks comfortable, but not very warm."
+	icon_state = "whiteout_robe"
+	item_state_slots = list(slot_r_hand_str = "suit_white", slot_l_hand_str = "suit_white")
+	flags_inv = HIDESHOES|HIDEJUMPSUIT|HIDETIE|HIDEHOLSTER

```

Comment:
Clear this line, please. Let it inherit directly from the parent type unless you specifically need to change something for the subtype.